checks/categories.py

```diff
@@ -2250,6 +2251,36 @@ def result_bad_redirect(self, tech_data):
         self.tech_data = tech_data
 
 
+class MailAuthTlsRptExists(Subtest):
+    def __init__(self):
+        super().__init__(
+            name="tlsrpt",
+            label="detail mail auth tlsrpt label",
+            explanation="detail mail auth tlsrpt exp",
+            tech_string="detail mail auth tlsrpt tech table",
+            worst_status=scoring.MAIL_AUTH_SPF_WORST_STATUS,
+            full_score=scoring.MAIL_AUTH_SPF_PASS,
+            model_score_field="tlsrpt_score",
+        )
+        # Fix for one line, one value data.
+        self.tech_data = [[self.tech_data]]
+
+    def result_good(self, tech_data):
+        self._status(STATUS_SUCCESS)
+        self.verdict = "detail mail auth tlsrpt verdict good"
+        self.tech_data = [[tech_data]]
+
+    def result_bad(self, tech_data):
+        self._status(STATUS_NOTICE)
+        self.verdict = "detail mail auth tlsrpt verdict bad"
+        if tech_data:
+            # More than one spf record. Show the records.
```

Comment:
I don't think this comment fits the rest of the code? Might be wrong even for SPF.